### PR TITLE
[WIP] First try of CMS idea

### PR DESCRIPTION
**Pull request by tw0517tw**

### CODE REVIEW

**Comment by LittleWhiteYA** [comment]
```js
const allActions = {
  helloWorld: {
    method: 'sendMessage',
    args: ['*Hello world*', { parse_mode: 'Markdown' } ],
  },
  photo: {
    method: 'sendPhoto',
    args: ['https://i.imgur.com/SQltmALl.jpg', {
      reply_markup: {
        inline_keyboard: [
          [ 
            { 
              text: 'Google',
              url: 'https://www.google.com',
            },
          ],
        ],
      },
      parse_mode: 'Markdown',
      disable_web_page_preview: false,
    },
    ]
  }
};

bot.use(context => {
  context.doActions = async action => {
    await context[action.method](...action.args);
  };
});

bot.onEvent(async context => {
  await context.doActions(allActions.helloWorld);
  await context.doActions(allActions.photo);
});
```

How about this?

File: examples/content-management/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# With Typing
+
+## Install and Run
+
+Download this example or clone [bottender](https://github.com/Yoctol/bottender).
+
+```
+curl https://codeload.github.com/Yoctol/bottender/tar.gz/master | tar -xz --strip=2 bottender-master/examples/with-typing
```
Comment (xxhomey19) [inline examples/content-management/README.md]: with-typing -> content-management

File: examples/content-management/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+cd with-typing
```
Comment (xxhomey19) [inline examples/content-management/README.md]: with-typing -> content-management

**Comment by chentsulin** [comment]
We need a lot of work to make this useful and it cannot be an official example anytime soon. 
So i am going to close this.

**Pull request closed without merging**